layouts/shortcodes/sign-language-section.html

```diff
@@ -0,0 +1,19 @@
+{{/*
+
+  Usage: {{% sign-language-section VIDEO_ID %}}
+
+  */ -}}
+
+{{ $youtube := "" -}}
+{{ $videoID := .Get 0 -}}
+{{ if $videoID -}}
+  {{ $youtube = printf "{{< youtube %s >}}" $videoID }}
+{{ else -}}
+  {{ warnf "%s: Shortcode 'sign-section' requires a video ID as argument" .Page.Path -}}
```

Comment:
Do you think we should handle the following warning due to the use of `.Page.Path` ? :)

```
WARN  .Path when the page is backed by a file is deprecated. We plan to use Path for a canonical source path and you probably want to check the source is a file. To get the current behaviour, you can use a construct similar to the one below:

  {{ $path := "" }}
  {{ with .File }}
        {{ $path = .Path }}
  {{ else }}
        {{ $path = .Path }}
  {{ end }}
```

Reply:
Yes, fixed, thanks.